src/components/Register/Register.tsx

```diff
@@ -0,0 +1,597 @@
+import React, { useEffect, useMemo } from 'react';
+import {
+  Button,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Stack,
+  TextField,
+  Typography
+} from '@mui/material';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { Controller, useForm } from 'react-hook-form';
+import { DatePicker, LocalizationProvider } from '@mui/lab';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import { ArrowRightAlt } from '@mui/icons-material';
+import FormHelperText from '@mui/material/FormHelperText';
+
+import './Register.css';
+import Side_Left from '../../img/Side_Left.png';
+
+interface FormData {
+  id: string;
+  email: string;
+  password: string;
+  name: string;
+  date: string;
+  gender: string;
+  province_id: number;
+  district_id: number;
+  ward_id: number;
```

Comment:
```suggestion
  province_id: number | string;
  district_id: number | string;
  ward_id: number | string;
```

Để support defaultValue="" cho Select